### PR TITLE
feat: add fonts

### PR DESCRIPTION
**Pull request by FrantisekMichalSebestyen**

### CODE REVIEW

File: resources/css/app.css [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import "oto-font.css";
```
Comment (igor-kamil) [inline resources/css/app.css:1]: [dust] I like the more generic approach. e.g. `typography`
fonts might change. but the file can remain.

File: resources/css/oto-font.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    font-weight: 300;
+    font-style: normal;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: "Oto Hand";
+    src: url("../../public/fonts/OtoHand-Regular.woff2") format("woff2");
+    font-weight: 300;
+    font-style: italic;
```
Comment (igor-kamil) [inline resources/css/oto-font.css:13]: sure? why not `normal`?

**Review by igor-kamil — APPROVED** [review]
LGTM

**Review by igor-kamil — COMMENTED** [review]
sorry I realised only now that fonts are not in `resources/fonts` but in `public/fonts`. can you fix it? (or I can do it)
https://laravel.com/docs/10.x/vite#blade-processing-static-assets

**Comment by FrantisekMichalSebestyen** [comment]
> sorry I realised only now that fonts are not in `resources/fonts` but in `public/fonts`. can you fix it? (or I can do it) https://laravel.com/docs/10.x/vite#blade-processing-static-assets

please do, i do not have time.

**Pull request merged**